Firestore/core/src/firebase/firestore/core/query.cc

```diff
@@ -28,10 +29,26 @@ namespace firebase {
 namespace firestore {
 namespace core {
 
+using Operator = Filter::Operator;
+using Type = Filter::Type;
+
 using model::Document;
```

Comment:
Nit: please move the `using` declarations into the unnamed namespace (now that there is one).